pkgs/development/python-modules/deltachat/default.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, buildPythonPackage
+, isPy27
+, fetchpatch
+, setuptools-scm
+, libdeltachat
+, cffi
+, IMAPClient
+, pluggy
+, setuptools
+, pytestCheckHook
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "deltachat";
+  inherit (libdeltachat) version src;
+  sourceRoot = "${src.name}/python";
```

Comment:
@link2xt Is it a problem that this gives version 1.52.0, which has not been released on PyPI?

Reply:
It's ok to always package the latest tagged version, there are no "bad" versions of libdeltachat.

Version 1.52 is not stable, it's one of the "release candidate" versions. We simply increase the second number whenever it looks like the version may be ready for release. Actual PyPI release will probably be done at the same time as android/ios/desktop releases.

Reply:
And will that actual release receive a new version? How can I tell release candidates and actual releases apart?

Reply:
1.52 is an actual core release, we just don't use all core releases for PyPI/Android/iOS/Desktop releases. There will likely be 1.53 or 1.54 for the next application release which should happen *very soon*™.

We don't have resources to maintain multiple branches and backport fixes, but all releases are stable and good enough for use thanks to lots of automated rust/python tests.

It's ok to always package the latest tagged core in NixOS.